setup.cfg

```diff
@@ -44,7 +44,7 @@ zip_safe = False
 dev =
     bumpver>=2023.1129
     pgtest~=1.3
-    pre-commit>=3.5
+    pre-commit>=4.0
```

Comment:
I think this version bump in the end was not needed, but probably doesn't hurt. Happy to revert.
(I still maintain we should not be installing pre-commit into every venv, it should instead be installed globally via `pipx` or `uv tool install`)